docs/Contribute/ContributeWrite/WriteDocs.md

```diff
@@ -0,0 +1,14 @@
+# Contribute to Documentation
+
+Helping us keeping the present [Documentation](https://github.com/btcpayserver/btcpayserver-doc) up-to-date is an important contribution because BTCPay Server evolves at each release.
+
+Beginners can view the following video on how to contribute to the BTCPay Server documentation:
+[![Contributing to Documentation](https://img.youtube.com/vi/bSDROcdSSWw/mqdefault.jpg)](https://www.youtube.com/watch?v=bSDROcdSSWw "How to contribute to BTCPay Server documentation")
+
+Contributing to documentation follows the same process as contributing for the Software Stack.
+
+**Step 1**
+
+Fork/Clone the main documentation repository ([BTCPay Server Documentation](https://github.com/btcpayserver/btcpayserver-doc)) using Github Desktop and publish it.
+
+**Then** follow **steps 2 through 5** of the [Software Stack](./WriteStack.md) contribution page that explains creating branches and modifying files with Guthub Desktop.
```

Comment:
I'm not a fan of pointing people to different page for the steps. Should we eliminate the written steps entirely since the video seems to be enough?

Reply:
I feel like the more ways we show (written + video is better than just written or just video ?) the better ?

Would it be suitable to just copy the writewtack.md steps into here (and the other docs) as to eliminate the link ?

Reply:
```suggestion
**Then** follow **steps 2 through 5** of the [Software Stack](./WriteStack.md) contribution page that explains creating branches and modifying files with Guthub.
```

Reply:
```suggestion
**Then** follow **steps 2 through 5** of the [Software Stack](./WriteStack.md) contribution page that explains creating branches and modifying files with Github.
```